tests/benchdnn/graph/setting_handler.cpp

```diff
@@ -314,6 +314,12 @@ bool get_binary_stag_and_dtag(const deserialized_op_t &base_op_ref,
         op_setting.stag = {{std::move(stag0), std::move(stag1)}};
     }
     op_setting.dtag.front() = std::move(dtag);
+
+    logical_tensor::dims src_strides = base_op_ref.in_lts_[0].stride_;
+    logical_tensor::dims wei_strides = base_op_ref.in_lts_[1].stride_;
+    const logical_tensor::dims &dst_strides = base_op_ref.out_lts_[0].stride_;
```

Comment:
Is it on purpose to only have `dst_strides` in const ref, while copy for `src_strides` and `wei_strides`?